e2e/test-util.ts

```diff
@@ -54,43 +80,75 @@ export async function navigateTo(page: Page, path: string) {
+  await page.getByRole("button", { name: "Create", exact: true }).click();
 
-  const nameInput = page.locator('#general-folder-storage vaadin-grid-cell-content').filter({ hasText: 'Name' }).locator('vaadin-text-field').nth(1).locator('input');
+  const nameInput = page
+    .locator("#general-folder-storage vaadin-grid-cell-content")
+    .filter({ hasText: "Name" })
+    .locator("vaadin-text-field")
+    .nth(1)
+    .locator("input");
   await nameInput.click();
   await nameInput.fill(folderName);
   await page.waitForSelector(`text=folder_open ${folderName}`);
+  await nameInput.fill("");
 }
 
 export async function deleteVFolderAndVerify(page: Page, folderName: string) {
-  await navigateTo(page, 'data');
-  const nameInput = page.locator('#general-folder-storage vaadin-grid-cell-content').filter({ hasText: 'Name' }).locator('vaadin-text-field').nth(1).locator('input');
+  await navigateTo(page, "data");
+  const nameInput = page
+    .locator("#general-folder-storage vaadin-grid-cell-content")
+    .filter({ hasText: "Name" })
+    .locator("vaadin-text-field")
+    .nth(1)
+    .locator("input");
   await nameInput.click();
   await nameInput.fill(folderName);
   await page.waitForTimeout(1000);
-  await page.getByRole('button', { name: 'delete' }).first().click();
-  await page.locator('#delete-without-confirm-button').getByLabel('delete').click();
-  await page.getByRole('tab', { name: 'delete' }).click();
+  await page.getByRole("button", { name: "delete" }).first().click();
+  await page
+    .locator("#delete-without-confirm-button")
+    .getByLabel("delete")
+    .click();
+  await page.getByRole("tab", { name: "delete" }).click();
 
   await page.waitForTimeout(1000);
-  const nameInputInTrash = page.locator('#trash-bin-folder-storage vaadin-grid-cell-content').filter({ hasText: 'Name' }).locator('vaadin-text-field').nth(1).locator('input');
+  const nameInputInTrash = page
+    .locator("#trash-bin-folder-storage vaadin-grid-cell-content")
+    .filter({ hasText: "Name" })
+    .locator("vaadin-text-field")
+    .nth(1)
+    .locator("input");
   await nameInputInTrash.click();
   await nameInputInTrash.fill(folderName);
-  await page.getByLabel('delete_forever').click();
-  await page.getByRole('textbox', { name: 'Type folder name to delete' }).fill(folderName);
   await page.waitForTimeout(1000);
-  await page.getByRole('textbox', { name: 'Type folder name to delete' }).click();
-  
-  await expect(page.locator('#trash-bin-folder-storage').getByText('e2e-test-folder', { exact: true })).toBeVisible();
-  
-  await page.getByRole('button', { name: 'Delete forever' }).click();
-  await expect(page.locator('#trash-bin-folder-storage').getByText('e2e-test-folder', { exact: true })).toBeHidden();
+  await page.locator("vaadin-grid-cell-content").filter({ hasText: folderName }).locator("//following-sibling::*[7]").locator(':visible').getByLabel('delete_forever').click();
```

Comment:
I have verified the test is successful, but the code is intermittently successful. It seems to be caused by not getting the updated folder list.

Reply:
Please check again. I use 'waitFor` before `fill` a input.

Reply:
Test still fails intermittently when running that code in VFolder sharing test